javascript-carousel/index.html

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>JavaScript Carousel</title>
+  <link
+    rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="row">
+      <div class="column chevron">
+        <i class="fas fa-chevron-left"></i>
+      </div>
+      <div data-view="1" class="column slide">
+        <div class="row image">
+          <img src="images/025.png" alt="">
+        </div>
+        <div class="row progress-dot">
+          <i data-view="1" class="fas fa-circle"></i>
+          <i data-view="2" class="far fa-circle"></i>
+          <i data-view="3" class="far fa-circle"></i>
+          <i data-view="4" class="far fa-circle"></i>
+          <i data-view="5" class="far fa-circle"></i>
+        </div>
+      </div>
+      <div data-view="2" class="column slide hidden">
+        <div class="row image">
+          <img src="images/039.png" alt="">
+        </div>
+        <div class="row progress-dot">
+          <i data-view="1" class="far fa-circle"></i>
+          <i data-view="2" class="fas fa-circle"></i>
+          <i data-view="3" class="far fa-circle"></i>
+          <i data-view="4" class="far fa-circle"></i>
+          <i data-view="5" class="far fa-circle"></i>
+        </div>
+      </div>
+      <div data-view="3" class="column slide hidden">
+        <div class="row image">
+          <img src="images/007.png" alt="">
+        </div>
+        <div class="row progress-dot">
+          <i data-view="1" class="far fa-circle"></i>
+          <i data-view="2" class="far fa-circle"></i>
+          <i data-view="3" class="fas fa-circle"></i>
+          <i data-view="4" class="far fa-circle"></i>
+          <i data-view="5" class="far fa-circle"></i>
+        </div>
+      </div>
+      <div data-view="4"  class="column slide hidden">
+        <div class="row image">
+          <img src="images/004.png" alt="">
+        </div>
+        <div class="row progress-dot">
+          <i data-view="1" class="far fa-circle"></i>
+          <i data-view="2" class="far fa-circle"></i>
+          <i data-view="3" class="far fa-circle"></i>
+          <i data-view="4" class="fas fa-circle"></i>
+          <i data-view="5" class="far fa-circle"></i>
+        </div>
+      </div>
+      <div data-view="5"  class="column slide hidden">
+        <div class="row image">
+          <img src="images/001.png" alt="">
+        </div>
+        <div class="row progress-dot">
+          <i data-view="1" class="far fa-circle"></i>
+          <i data-view="2" class="far fa-circle"></i>
+          <i data-view="3" class="far fa-circle"></i>
+          <i data-view="4" class="far fa-circle"></i>
+          <i data-view="5" class="fas fa-circle"></i>
+        </div>
+      </div>
```

Comment:
Instead of having a separate view container for each image, I would have one container with dots and arrow icons, and just have all your image elements in one container, and hide/show those elements